azure-pipelines/secure-development-tools.yml

```diff
@@ -1,5 +1,14 @@
 steps:
 
+ ### Scan binaries with Windows Defender
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
+  displayName: 'Scan with Windows Defender'
+  inputs:
+    TreatSignatureUpdateFailureAs: Warning
+    SignatureFreshness: OneDay
+    FileDirPath: '$(Build.SourcesDirectory)/bin/'
+  continueOnError: True
```

Comment:
Why scan and only emit a warning on failure? Those will typically go unnoticed. Is it because of a high false positive rate?

Reply:
I will remove this.